src/site/shared/tests/helpers/GetToggles.ts

```diff
@@ -0,0 +1,15 @@
+import {Matcher, screen} from "@testing-library/react";
+
+/**
+ * Gets the ButtonToggle or SwitchToggle image elements associated with the provided text.
+ * The pressed image will be returned first and the unpressed second.
+ *
+ * @param id The id of the text to search for.
+ * @returns    An array containing the pressed and unpressed images, respectively.
+ */
+export function GetToggles(id: Matcher): [HTMLImageElement, HTMLImageElement] {
+    const buttons = screen.getByText(id).parentNode!.querySelectorAll("img");
+    return buttons[0].src.includes("Down")
+        ? [buttons[0], buttons[1]]
+        : [buttons[1], buttons[0]];
+}
```

Comment:
You are pretty much strictly using this helper to then check to make sure that the Toggle is on/off, you should probably instead just have a helper that gets the state of the toggle, and then `expect` that state to be on/off. Then in the actual ButtonToggle/SwitchToggle tests, that's where the functionality of the images being correspondingly on/off should be tested

Reply:
What would I use to check if it on/off? I would think it would be checking if the "down" button is visible or not, but the `toBeVisible` thing I've been using to test checks a few different conditions: https://github.com/testing-library/jest-dom#tobevisible

Reply:
If the expected image is visible or not yeah